### DISCUSSION TITLE
Upgrading from Next v14.2.7 to v14.2.8 Broke Some Server Actions in Production on Vercel

### DISCUSSION THREAD

**Discussion by manavm1990** [vercel/next.js#69761] [App Router] [answered]
[logs_result.json](https://github.com/user-attachments/files/16900771/logs_result.json)

It was all working fine locally. I investigated the logs and saw this on Vercel. I knew that we had not made any code updates to justify these.

After I did: `npm install next@14.2.7`, the issue seemed to have been resolved? 

Happy to provide additional details if needed. Baffling...

**Comment by Juneezee** [comment, marked as answer]
There are other users experiencing the same issue. See https://github.com/vercel/next.js/issues/69756
